src/Tizen.NUI.Components/Controls/Button.Internal.cs

```diff
@@ -289,6 +289,11 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
         {
             Debug.Assert(size != null);
 
+            if (size.Equals(this.size))
+            {
+                return;
+            }
+
```

Comment:
same as above. (warning at line 308)
null checking of `controlStateChangedInfo`

Reply:
Done.